src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -979,24 +979,14 @@ void CouplingSchemeConfiguration::addDataToBeExchanged(
     const bool requiresInitialization = exchange.requiresInitialization;
     if (from == accessor) {
       scheme.addDataToSend(exchange.data, exchange.mesh, requiresInitialization);
-      if (requiresInitialization && (_config.type == VALUE_SERIAL_EXPLICIT || _config.type == VALUE_SERIAL_IMPLICIT)) {
-        if (not _experimental) {
-          PRECICE_CHECK(not scheme.doesFirstStep(),
-                        "In serial coupling only second participant can initialize data and send it. "
-                        "Please check the <exchange data=\"{}\" mesh=\"{}\" from=\"{}\" to=\"{}\" initialize=\"{}\" /> tag in the <coupling-scheme:... /> of your precice-config.xml. You can bypass this restriction by activating the experimental interface for time interpolation. Please refer to the documentation for more details.",
-                        dataName, meshName, from, to, requiresInitialization);
-        }
-      }
```

Comment:
We only define the waveform-order on the read-data. Therefore, we cannot check it here and raise a warning. Meaning: The first participant will not raise a warning.